### PR TITLE
Replace Install/Download Engine Links

### PR DESCRIPTION
**Pull request by ricardobaeta**
Signed-off-by: Ricardo Baeta <ricardo@ricardobaeta.com>

### CODE REVIEW

File: hugo/layouts/partials/header.html [hunk 1]
```diff
@@ -54,7 +54,7 @@
                         <a class="btn btn-sm btn-highlight-light" href="https://go.sourced.tech/engine">Request a Demo</a>
                     </li>
                     <li class="nav-item nav-btn">
-                        <a class="btn btn-sm btn-highlight" href="https://docs.sourced.tech/engine/#quickstart">Install Engine</a>
+                        <a class="btn btn-sm btn-highlight" href="https://go.sourced.tech/engine-download">Install Engine</a>
```
Comment (dpordomingo) [inline hugo/layouts/partials/header.html]: shouldn't we use the product name?
I was said that we should use `source{d} Engine` always, with no exception.

File: hugo/layouts/partials/try-sourced.html [hunk 2]
```diff
@@ -7,7 +7,7 @@ <h2 class="text-white mt-5">Try source{d} Engine today</h2>
         <div class="col-lg-6 offset-lg-3 mt-4 mb-5">
             <div class="row">
                 <a class="btn btn-highlight-light btn-lg col-sm mr-3" href="https://go.sourced.tech/engine">Request a Demo</a>
-                <a class="btn btn-highlight btn-lg col-sm" href="https://docs.sourced.tech/engine/#quickstart">Install Engine</a>
+                <a class="btn btn-highlight btn-lg col-sm" href="https://go.sourced.tech/engine-download">Install Engine</a>
```
Comment (dpordomingo) [inline hugo/layouts/partials/try-sourced.html]: what about using `source{d} Engine`?

Comment (dpordomingo) [reply hugo/layouts/partials/try-sourced.html]: Should we change this other to be `source{d} Engine`?

File: hugo/layouts/engine/single.html [hunk 3]
```diff
@@ -259,7 +259,7 @@ <h3 class="title-line text-dark line-green">source{d} Engine in five minutes</h3
                             <div class="col-lg-9">
                                 <h3>Download source{d} Engine</h3>
                                 <p>Leverage the source{d} Engine to turn your source code across versions into actionable data & business intelligence.</p>
-                                <a href="https://docs.sourced.tech/engine#quickstart" class="btn btn-highlight-light">Get Started</a>
+                                <a href="hhttps://docs.sourced.tech/engine#quickstart" class="btn btn-highlight-light">Get Started</a>
```
Comment (dpordomingo) [inline hugo/layouts/engine/single.html]: Typo here.
```suggestion
                                <a href="https://docs.sourced.tech/engine#quickstart" class="btn btn-highlight-light">Get Started</a>
```

**Comment by ricardobaeta** [comment]
You're correct @dpordomingo ! And on a separate PR, which is not as important as this one because:

image

File: hugo/layouts/engine/single.html [hunk 4]
```diff
@@ -7,7 +7,7 @@
             <br><br>
             <h1 class="title text-left p-0 mt-5 mb-4">Turn your code into actionable insights with source{d} Engine</h1>
             <a href="https://go.sourced.tech/engine" class="btn btn-purple-light mr-4 mb-3 mb-sm-5">Request A Demo</a>
-            <a href="https://docs.sourced.tech/engine#quickstart" class="btn btn-highlight-light mb-3 mb-sm-5">Install source{d} Engine</a>
+            <a href="https://go.sourced.tech/engine-download" class="btn btn-highlight-light mb-3 mb-sm-5">Install source{d} Engine</a>
```
Comment (marnovo) [inline hugo/layouts/engine/single.html]: We can cut the source{d} from the button caption (`Install Engine` is fine), as in this particular case it makes the CTA disproportionately long here and you have a ton of "source{d}" and a few "Engine" around the navbar/hero/header so it's not something you'd miss.

File: hugo/layouts/partials/header.html [hunk 5]
```diff
@@ -54,7 +54,7 @@
                         <a class="btn btn-sm btn-highlight-light" href="https://go.sourced.tech/engine">Request a Demo</a>
                     </li>
                     <li class="nav-item nav-btn">
-                        <a class="btn btn-sm btn-highlight" href="https://docs.sourced.tech/engine/#quickstart">Install Engine</a>
+                        <a class="btn btn-sm btn-highlight" href="https://go.sourced.tech/engine-download">Install source{d} Engine</a>
```
Comment (marnovo) [inline hugo/layouts/partials/header.html]: Same as above review comment. The context around it makes for a very strong and obvious connection the engine.

File: hugo/layouts/partials/try-sourced.html (hunk 2 above)
Comment (marnovo) [reply hugo/layouts/partials/try-sourced.html]: See my previous comments.

**Comment by marnovo** [comment]
As for what @dpordomingo mentioned on documentation:

- We have the link in Docs in the navbar
- We have it on the open source page
- We don't (but could) have it linked in the https://go.sourced.tech/engine-download page so that people who don't want to give their email still can find the documentation/repo during the user journey.

File: hugo/layouts/engine/single.html (hunk 4 above)
Comment (ricardobaeta) [reply hugo/layouts/engine/single.html]: Thanks @marnovo ! Done.

File: hugo/layouts/partials/header.html (hunk 5 above)
Comment (ricardobaeta) [reply hugo/layouts/partials/header.html]: Thanks @marnovo ! Done.

**Comment by ricardobaeta** [comment]
@marnovo @dpordomingo Sorry guys, but I've accidentally addressed Marcelo's requested changes on this PR commit https://github.com/src-d/landing/pull/360/commits/7800ac8851095a6c98b76d2b765235f26f0d1600

**Comment by marnovo** [comment]
Regarding the discussion on [supported resolutions & browsers](https://github.com/src-d/landing/pull/359#issuecomment-462337478), [please use this reference](https://github.com/src-d/guide/issues/266#issuecomment-455314000) (we still need to PR it to the guide I guess).

**Comment by dpordomingo** [comment]
I restarted Drone.
If it pass, I'll merge and deploy as requested by @ricardobaeta

**Pull request merged**